js/src/phemex.js

```diff
@@ -281,6 +281,7 @@ export default class phemex extends Exchange {
             'exceptions': {
                 'exact': {
                     // not documented
```

Comment:
@phips28 .JS files don't need to be committed they're automatically generated.  🙂

Reply:
do you need a new PR? or does it work even if I have changed the JS file?

Reply:
I reverted it don't worry